.pipelines/cni/k8s-e2e/k8s-e2e-job-template.yaml

```diff
@@ -0,0 +1,166 @@
+            attempts: 3
+      - ${{ if eq(parameters.liveness, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Liveness Probes
+            name: liveness
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '\[sig-node\].Probing'
+            ginkgoSkip: ''
+            os: ${{ parameters.os }}
+            processes: 4
+            attempts: 10
+      - ${{ if eq(parameters.portforward, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Kubectl Portforward
+            name: portforward
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '\[sig-cli\].Kubectl.Port'
+            ginkgoSkip: ''
+            os: ${{ parameters.os }}
+            processes: 4
+            attempts: 3
+      - ${{ if eq(parameters.dsNodeRestart, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Node Restart
+            name: restart
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '\[sig-apps\].DaemonRestart'
+            ginkgoSkip: ''
+            os: ${{ parameters.os }}
+            processes: 4
+            attempts: 3
+      - ${{ if eq(parameters.token, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Service Account
+            name: token
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '\[sig-auth\].ServiceAccounts'
+            ginkgoSkip: ''
+            os: ${{ parameters.os }}
+            processes: 4
+            attempts: 3
+      - ${{ if eq(parameters.loadBalancer, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Load Balancers
+            name: load
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '\[sig-network\].LoadBalancers'
+            ginkgoSkip: 'ESIPP|Serial'
+            os: ${{ parameters.os }}
+            processes: 4
+            attempts: 3
+      - ${{ if eq(parameters.metrics, true) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Metrics Server
+            name: metrics
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '\[sig-instrumentation\].MetricsGrabber'
+            ginkgoSkip: ''
+            os: ${{ parameters.os }}
+            processes: 4
+            attempts: 3
+      - ${{ if and(eq(parameters.hybridWin, true), eq(parameters.os, 'windows')) }}:
+        - template: ../k8s-e2e/k8s-e2e-step-template.yaml
+          parameters:
+            testName: Hybrid Network
+            name: hybrid
+            clusterName: ${{ parameters.clusterName }}
+            ginkgoFocus: '\[sig-windows\].Hybrid'
+            ginkgoSkip: ''
```

Comment:
where are we running service conformace and host port conformance tests?

Reply:
We are not currently running either of those. I can add them in.